apps/service_providers/llm_service/runnables.py

```diff
@@ -185,7 +184,8 @@ def _build_chain(self) -> Runnable[dict[str, Any], Any]:
 
     @property
     def prompt(self):
-        system_prompt = SystemMessagePromptTemplate.from_template(self.experiment.prompt_text)
+        prompt = self.experiment.prompt_text + "\nCurrent datetime: {current_time}"
```

Comment:
While this is in my opinion not the most elegant solution, I can't think of another one except using a human message. I don't think we should use a human message for this though.

Reply:
No necessary to change right now but I think we should expand this a bit more.

```
Today's datetime on UTC time 2023-05-02T10:00:00+00:00, it's Tuesday.
```